spec/adhearsion/call_spec.rb

```diff
@@ -614,13 +619,21 @@ module Adhearsion
           expect(Adhearsion.active_calls.size).to eq(size_before)
         end
 
-        it "shuts down the actor" do
+        it "shuts down the actor using platform config" do
           Adhearsion.config.platform.after_hangup_lifetime = 2
           subject << end_event
           sleep 2.1
           expect(subject.alive?).to be false
           expect { subject.id }.to raise_error Call::ExpiredError, /expired and is no longer accessible/
         end
+
+        it "shuts down the actor using the call after_hangup_lifetime instance" do
+          subject.after_hangup_lifetime = 2
```

Comment:
It strikes me that this test may be leaky. The above test sets global config, but I don't see that config being unset before attempting to use the per-call config here.